Modelica/Magnetic/QuasiStatic/FluxTubes/Shapes/FixedShape/HollowCylinderRadialFlux.mo

```diff
@@ -13,14 +13,14 @@ model HollowCylinderRadialFlux
     annotation (Dialog(group="Fixed geometry"));
   parameter SI.Radius r_o=0.02 "Outer radius of hollow cylinder"
     annotation (Dialog(group="Fixed geometry"));
-  parameter SI.Angle alpha=2*Modelica.Constants.pi
-  "Central angle";
+  parameter SI.Angle alpha=2*pi "Central angle";
 equation
-  A = l*pi*(r_o + r_i);
+  A = (r_o + r_i)/2*alpha*l;
   // Area at arithmetic mean radius for calculation of average flux density
-  G_m = 2*pi*mu_0*mu_r*l/Modelica.Math.log(r_o/r_i)*alpha/(2*Modelica.Constants.pi);
+  G_m = mu_0*mu_r*alpha*l/Modelica.Math.log(r_o/r_i);
```

Comment:
This is a bug fix, as the angle `alpha` was not considered correctly until now.

Reply:
So if you think this is important enough to include a hypothetical 4.0.1 release, then this part should probably be moved to its own PR which can then also be backported into the 4.0.x branch.